mllib/src/main/scala/org/apache/spark/ml/feature/StandardScaler.scala

```diff
@@ -29,7 +29,22 @@ import org.apache.spark.sql.types.{StructField, StructType}
 /**
  * Params for [[StandardScaler]] and [[StandardScalerModel]].
  */
-private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol
+private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol {
+  
+  /**
+   * Whether to center the data before scaling
+   * @group param
+   */
+  val withMean: BooleanParam = new BooleanParam(this, "withMean", "Center data with mean")
+  
+  /**
+   * Whether to scale the data to have unit standard deviation
+   * @group param
+   */
+  val withStd: BooleanParam = new BooleanParam(this, "withStd", "Scale to unit standard deviation")
+}
+
+
```

Comment:
Remove extra newline  (generally 1 blank line between classes, though I think 2 is OK too)